packages/components/src/CircularProgress/CircularProgress.test.js

```diff
@@ -19,6 +19,11 @@ describe('CircularProgress', () => {
 		expect(wrapper).toMatchSnapshot();
 	});
 
+	it('should render at larger size if set', () => {
+		const wrapper = renderer.create(<CircularProgress size="larger" />).toJSON();
```

Comment:
`larger` should come from the `SIZE.larger` constant.

Reply:
fixed